client/src/Authorization/_auth-services/authentication.js

```diff
@@ -51,6 +51,7 @@ const reset = (email) => {
 
     let myHeaders = new Headers();
     myHeaders.append("Content-Type", "application/json");
+    myHeaders.append("Content-Type", "text/plain");
```

Comment:
Is it required ? Or it's from previous version ?

Reply:
It's from the previus version. I'll correct it ASAP.